src/main/webapp/EditProfilePage.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<body>
+	<h1>Edit Your Profile Page</h1>
```

Comment:
The google style guidelines that we are following say that for HTML pages the default indentation space is 2 instead of 4. https://google.github.io/styleguide/htmlcssguide.html#Indentation